### ISSUE TITLE
Cannot read property 'mq' of undefined

### ISSUE THREAD

**Issue by skovy** [pedronauck/docz#512]
## Bug Report

**Describe the bug**

When upgrading `docz-theme-default` to the most recent version ([`v0.12.16`](https://github.com/pedronauck/docz/releases/tag/v0.12.16)), the error `Cannot read property 'mq' of undefined` is being thrown. 

**To Reproduce**

1. Install the most recent version
    - `docz-theme-default@0.12.16`
    - `docz@0.12.16`
2. Run `yarn docz:dev`
3. Open the docs

```shell
🚨 Whoops, we crashed 🚨
Cannot read property 'mq' of undefined
Stack trace
in div (created by Context.Consumer)
in div (created by Context.Consumer)
in Main (created by Context.Consumer)
in ThemeConfig (created by Page)
in Page (created by AsyncRoute)
in AsyncRoute (created by Route)
in Route (created by Context.Consumer)
in Switch (created by Context.Consumer)
in MDXProvider (created by DocPreview)
in DocPreview (created by Context.Consumer)
in ThemeProvider (created by Context.Consumer)
in ThemeConfig (created by Theme)
in Theme (created by Theme)
in ScrollToTopBase (created by Route)
in Route (created by withRouter(ScrollToTopBase))
in withRouter(ScrollToTopBase) (created by Theme)
in Router (created by BrowserRouter)
in BrowserRouter (created by Router)
in Router (created by Theme)
in DataServer (created by Theme)
in StateProvider (created by Theme)
in ErrorBoundary (created by Theme)
in Theme (at root.jsx:6)
in Root (created by HotExportedRoot)
in AppContainer (created by HotExportedRoot)
in HotExportedRoot (at app/index.jsx:15)
```

**Specific Code Erroring**
https://github.com/pedronauck/docz/blob/4b3cb327403dfb8a5fa9c79247c36db27556815a/packages/docz-theme-default/src/components/shared/Sidebar/index.tsx#L28-L31

**Expected behavior**

The docs should render.

**Environment**

* `masOC@v10.13.4`
* `yarn@v1.9.4`
* `node@v10.8.0`

**Additional context/Screenshots**
- I'm assuming it's related to https://github.com/pedronauck/docz/pull/490/files
- I'm assuming it's related to https://github.com/pedronauck/docz/issues/452 but I believe this is a different issue as that one was resolved in the most recent release.

**Comment by pedronauck** [comment]
I trying a lot of things on this but I think that is something related to emotion-theming when using react@16.7.0-alpha.*, because I made a lot of tests using react@16.6.3 and everything it's working fine.

**Comment by pedronauck** [comment]
Fixed on [v0.12.7](https://github.com/pedronauck/docz/releases/tag/v0.12.17). Please, add `docz-theme-default` as well as a dependency if you don't have it explit on your package.json 🙏

**Comment by skovy** [comment]
Thanks for the speedy turnaround @pedronauck!

I now realize I forgot to mention the version of React, it was happening with  `react@16.5.2` and `react-dom@16.5.2` but `v0.12.16` works after upgrading `react`/`react-dom` to `16.6.3`.
